tests/integration/test_charm.py

```diff
@@ -41,3 +42,27 @@ async def test_reachable(app: Application):
     response = requests.get(f"http://{ip_address}:{port}")
 
     assert response.status_code == 200
+
+
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_ingress_reachable(app: Application):
+    """
+    arrange: given charm has been built, deployed and related to a dependent application
+    act: when the dependent application is queried via the ingress
+    assert: then the response is HTTP 200 OK.
+    """
+    # Get the IP address which is in the status message
+    ip_regex = r"[0-9]+(?:\.[0-9]+){3}"
+    ip_address_match = re.findall(ip_regex, app.units[0].workload_status_message)
+    assert (
+        ip_address_match
+    ), f"could not find IP address in status message: {app.units[0].workload_status_message}"
+    ip_address = ip_address_match[0]
+    port = "80"
+    ps = subprocess.Popen(['echo', f'{ip_address} hello-kubecon'], stdout=subprocess.PIPE)  # nosec
```

Comment:
Does this pass the linter? I would expect this to be enquoted in double quotes

Reply:
yes, it passes the linter and executes the task correctly

Reply:
Yes, but when changing it to the linter we use in indico, I get a lot of errors that are out of the scope of this PR. I'd suggest to open a new PR with those changes and address them there.

Reply:
Ok. Can you change those quotes in the interim? I'd be better to keep it as compliant with the other linter as possible

Reply:
Sure, no problem